main/main.go

```diff
@@ -149,54 +149,32 @@ func getPaths(path string, c component) (fileAndBucketPaths, error) {
 	}
 }
 
-func getK8sAndIstioCVEs(feedReader map[string]string) ([]utils.NvdCVEEntry, []utils.NvdCVEEntry, error) {
-	var allK8sCVEs, allIstioCVEs []utils.NvdCVEEntry
+func getK8sAndIstioCVEs(feedReader map[string]string) ([]nvd.CVEEntry, []nvd.CVEEntry, error) {
+	var allK8sCVEs, allIstioCVEs []nvd.CVEEntry
 
 	for feedName, fileName := range feedReader {
 		dat, err := ioutil.ReadFile(fileName)
 		if err != nil {
 			return nil, nil, errors.Wrapf(err, "error reading file %q", fileName)
 		}
 
-		cves, err := utils.GetCVEs(dat)
+		cves, err := nvd.Load(dat)
 		if err != nil {
 			return nil, nil, errors.Wrapf(err, "error unmarshalling file %q", fileName)
 		}
 
-		k8sCVEs := filterCVEs(cves, "kubernetes")
+		k8sCVEs := cves.Filter(nvd.NewCPEMatcher("kubernetes:kubernetes"))
```

Comment:
This seems a bit... weird to me. The CPEMatcher feels like it should abstract away details of CPEs, and yet we are constructing a substring here with CPE-style formatting. The old way (where the method accepted a package name, and handled the logic of creating the substring) made more sense to me.